### PR TITLE
Update Mysqli::construct() docs for return value as of PHP 8.1

### PR DESCRIPTION
**Pull request by mattkolb**
Docs need updating according to this:
https://www.php.net/manual/en/migration81.incompatible.php#migration81.incompatible.mysqli

### CODE REVIEW

File: reference/mysqli/mysqli/construct.xml [hunk 1]
```diff
@@ -132,7 +132,7 @@
    &return.falseforfailure;.
   </para>
   <para>
-   <methodname>mysqli::connect</methodname> returns &null; on success&return.falseforfailure;.
+   <methodname>mysqli::connect</methodname> returns &null; on success&return.falseforfailure;. As of PHP 8.1, returns true on success.
```
Comment (kamil-tekiela) [inline reference/mysqli/mysqli/construct.xml:135]:
```suggestion
   <methodname>mysqli::connect</methodname> returns &null; on success&return.falseforfailure;. As of PHP 8.1, returns &true; on success.
```

**Review by Girgias — CHANGES_REQUESTED** [review]
This is missing a Changelog section.

File: reference/mysqli/mysqli/construct.xml [hunk 2]
```diff
@@ -159,6 +160,12 @@
         All parameters are now nullable.
        </entry>
       </row>
+      <row>
+       <entry>8.1.0</entry>
+       <entry>
+        <methodname>mysqli::connect</methodname> now returns &true; instead of &null; on success.
+       </entry>
+      </row>
```
Comment (Girgias) [inline reference/mysqli/mysqli/construct.xml:168]: Must be above the 7.4.0 change log entry

File: reference/mysqli/mysqli/construct.xml [hunk 3]
```diff
@@ -133,6 +133,7 @@
   </para>
   <para>
    <methodname>mysqli::connect</methodname> returns &null; on success&return.falseforfailure;.
+   As of PHP 8.1.0, returns &true; on success.
```
Comment (Girgias) [inline reference/mysqli/mysqli/construct.xml:136]:
```suggestion
   <methodname>mysqli::connect</methodname> &return.success;.
```
Is what I want to use, but the entity starts with an uppercase letter. :-/

Also, the "return value" for ``__construct()`` doesn't really make sense looking at it

**Review by Girgias — CHANGES_REQUESTED** [review]

File: reference/mysqli/mysqli/construct.xml (hunk 3 above)
Comment (kamil-tekiela) [reply reference/mysqli/mysqli/construct.xml:132]: I don't think I can use this entity here. This page should probably be split into 3 separate pages. For now, I would suggest to keep it as it is in the PR.

Comment (Girgias) [reply reference/mysqli/mysqli/construct.xml:132]: Possibly, or maybe just into 2 where the constructor is seperate.

But I would use more the wording:

```xml
   <methodname>mysqli::connect</methodname> returns &true; on success&return.falseforfailure;.
   Prior to PHP 8.1.0, returns &null; on success.
```

**Review by Girgias — APPROVED** [review]

**Pull request merged**